### PR TITLE
Added redux and some examples

### PR DESCRIPTION
**Pull request by YoniX44**
all jail jhon!

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { configureStore } from "@reduxjs/toolkit";
+import counterReducer from "./features/counter/counterSlice";
+import jhonReducer from "./features/jhon/jhonSlice"
+
+export default configureStore({
+  reducer: {
+    counter: counterReducer,
+    jhon: jhonReducer,
+
```
Comment (idobarlev) [inline src/store.js:9]: Delete line

File: src/features/jhon/jhonSlice.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const jhonSlice = createSlice({
+  name: "jhon",
```
Comment (idobarlev) [inline src/features/jhon/jhonSlice.js:4]: Where do you use name?

File: src/features/jhon/jhonSlice.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  initialState: {
+    value: "",
+  },
+  reducers: {
+    addJhon: (state) => {
+      state.value = state.value + "Jhon";
```
Comment (idobarlev) [inline src/features/jhon/jhonSlice.js:10]: Use name here?

File: src/store.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { configureStore } from "@reduxjs/toolkit";
+import counterReducer from "./features/counter/counterSlice";
+import jhonReducer from "./features/jhon/jhonSlice"
+
+export default configureStore({
+  reducer: {
+    counter: counterReducer,
+    jhon: jhonReducer,
```
Comment (idobarlev) [inline src/store.js:8]: Why the files are called slicers and not reducers?

File: src/Containers/Home.js [hunk 5]
```diff
@@ -1,13 +1,52 @@
 import React from "react";
 import { Typography } from "@material-ui/core";
 import Navbar from "../Components/Navbar";
+import { useSelector, useDispatch } from "react-redux";
+import { decrement, increment } from "../features/counter/counterSlice";
+import { addJhon, removeJhon } from "../features/jhon/jhonSlice";
 
 const Home = ({ user }) => {
+  const count = useSelector((state) => state.counter.value);
```
Comment (idobarlev) [inline src/Containers/Home.js:9]: Separate the selectors to different files

**Review by idobarlev — COMMENTED** [review]
Fix comments pls, try to separate selectors and import the needed actions on Home.

File: src/features/jhon/jhonSlice.js (hunk 3 above)
Comment (YoniX44) [reply src/features/jhon/jhonSlice.js:10]: yes

File: src/store.js (hunk 4 above)
Comment (YoniX44) [reply src/store.js:8]: because the redux toolkit is called slicers for some reason :S

**Pull request merged**